src/internal/connector/data_collections.go

```diff
@@ -198,7 +200,7 @@ func checkServiceEnabled(
 		canMakeDeltaQueries = info.CanMakeDeltaQueries()
 	}
 
-	return true, canMakeDeltaQueries, nil
```

Comment:
very minor: imo this intersection should occur when handling the result of checkServiceEnabled, instead of passing down the opts into this call.  It's not much difference, but it does keep concerns separate.  The concern of this func is to report on the results of looking up the resource owner info.  The concern of the caller is to evaluate the combination of passed in options and lookup results.

Reply:
This was moved out. Looks like I missed rebased with that commit in this PR.